ansible/localhost-setup-ubuntu20.04-devpc.yml

```diff
@@ -13,9 +13,10 @@
         prompt: "[Warning]: Skip install GPU modules. Try manually installing these modules to execute nodes depending on TensorRT."
```

Comment:
You should comment out the prompt to decide if GPU should be installed because it won't be installed as of now anyhow

Reply:
Comment out these GPU-related tasks.